### ISSUE TITLE
Ecosystem alt text is redundant

### ISSUE THREAD

**Issue by oraNod** [ansible-community/community-website#233]
Ensure that the alternative text for each image or image button is appropriate while removing unnecessary redundancy. If the content of the image is already conveyed elsewhere (through text or the alternative text of a nearby image) or if the image does not convey content, the image may generally be given empty/null alternative text (alt=""). Image buttons always convey a specific function, and thus cannot be given null alternative text.
